dydx/dydxPresenters/dydxPresenters/_v4/TakeProfitStopLoss/dydxTakeProfitStopLossViewPresenter.swift

```diff
@@ -187,7 +183,8 @@ private class dydxTakeProfitStopLossViewPresenter: HostedViewPresenter<dydxTakeP
         }
 
         if let error = errors.first {
-            viewModel?.submissionReadiness = .fixErrors(cta: error.resources.action?.localized)
+            let errorCta = error.resources.action?.localized ?? DataLocalizer.shared?.localize(path: error.resources.action?.stringKey ?? "", params: nil)
```

Comment:
There is an extension to use:

`
extension ErrorString {
    var localizedString: String? {
        localized ?? DataLocalizer.localize(path: stringKey, params: params?.dictionary)
    }
}
`

Reply:
addressed d8051ac4f88e897f79c6c21abeeb9533841a8e51